Cython/Compiler/Nodes.py

```diff
@@ -1887,8 +1888,13 @@ class FuncDefNode(StatNode, BlockNode):
     #       by AnalyseDeclarationsTransform, so it can replace CFuncDefNodes
     #       with fused argument types with a FusedCFuncDefNode
 
+    class NeedsClosure(enum.IntEnum):
```

Comment:
This stuff is copied fairly directly from https://github.com/cython/cython/pull/6226 since it's useful here.